packages/cli/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # API Elements: CLI Changelog
 
+## 0.10.3 (2019-08-31)
```

Comment:
No strong opinions here, although we don't necessarily need to release the CLI as these changes have no impact on it. The new version of the apib serialize adapter is also compatible with the existing version `"@apielements/apib-serializer": "^0.16.1",` so installing CLI would get updated apib-serializer regardless.

Reply:
Right. I saw similar releases on the past, that's why I did it.

Reply:
The past ones are usually a slightly different case where the version range didn't suffice (or core changed). Such as minor releases which don't fit into the patch range (0.17 for example won't fit into that range, and might trigger CLI release).

Reply:
In that case I'll drop this release.

Reply:
Dropped in e0d00c3